dockerfiles/init/modules/openshift/files/scripts/multi-user/deploy_postgres_only.sh

```diff
@@ -9,67 +9,15 @@
 COMMAND_DIR=$(dirname "$0")
 export CHE_EPHEMERAL=${CHE_EPHEMERAL:-false}
 
-oc apply -f "$COMMAND_DIR"/che-init-image-stream.yaml
+IMAGE_POSTGRES=${IMAGE_POSTGRES:-"eclipse/che-psql:nightly"}
```

Comment:
I think we have to change `che-psql`  name, because `psql` is CLI tool for managing PostgreSQL DB, that may confuse.